docs/general/polkadot-uc.md

```diff
@@ -0,0 +1,337 @@
+---
+id: polkadot-uc
+title: The Polkadot Ubiquitous Computer
+sidebar_label: Ubiquitous Computer
+description: Polkadot as a Decentralized and Secure Ubiquitous Computer.
+keywords:
+  [polkadot vison, polkadot 2.0, Decentralization, ubiquitous computer, coretime, blockspace]
+slug: ../polkadot-uc
+---
+
+:::info
+
+The material on this page is based on
+[Gavin Wood's talk at Polkadot Decoded 2023](https://www.youtube.com/watch?v=GIB1WeVuJD0).
+
+:::
+
+Understanding what [Polkadot 1.0](./polkadot-v1.md) is about and the philosophy behind it will help
+us to envision the future direction of the Polkadot ecosystem toward abstraction and generalization.
+
+## Polkadot as a Computational Resource
+
+Polkadot has been abstracted and generalized beyond what was originally proposed and envisioned in
+the [whitepaper](https://polkadot.network/whitepaper/). Polkadot is:
+
+- About [**Blockspace**](./polkadot-v1.md#polkadots-blockspace) (the underlying resources that
+  chains need), not chains.
+- **A platform to build applications** rather than chains, and for people to use those applications.
+  Fundamentally, Polkadot is not a platform to host chains, and so far, chains happened to be one
+  way to build applications and grow Polkadot's utility.
+- **A provider of resilient general-purpose continuation computation**, where the term
+  _continuation_ refers to a broad long-running task that can do something, pause, continue (or do
+  something else) later.
+- **A multicore computer** where chains that continuously operate in parallel on different cores are
+  called [parachains](../learn/learn-parachains.md). Currently, one core is reserved for one chain
+  through [slot auction mechanism](../learn/learn-auction.md), although one core can be reserved
+  on-demand to multiple chains at different periods (see
+  [parathreads](../learn/learn-parathreads.md)). At the time of writing (mid 2023), on Polkadot,
+  there are around 50 cores independently operating in parallel. Prediction models show that cores
+  can increase to 500-1000 in the following few years.
+
+As a computational resource, Polkadot can be described as a traditional CPU, and at the time of
+writing it has the following benchmarking features:
+
+- Bandwidth: ~1MB/s
+- Compute: Geekbench 5 SC score ~380
+- Latency: 6s (time between two consecutive tasks)
```

Comment:
i think you should provide sources and reproducible benchmarks for this before putting such things in docs

Reply:
I thought these data were already coming from some tests (as they were presented at the Decoded)

Reply:
sounds like you should check this before you write down specific benchmarks into a polkadot official wiki

Reply:
we should verify that and reference it

Reply:
deleted part until we verify those data